rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleConnection.java

```diff
@@ -417,4 +422,23 @@ public String toString() {
      */
     @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
     Observable<Integer> requestMtu(int mtu);
+
+    /**
+     * <b>This method requires deep knowledge of RxAndroidBLE internals. Use it only as a last resort if you know
+     * what your are doing.</b>
+     * <p>
+     * Queue an operation for future execution. The method accepts a {@link RxBleRadioOperationCustom} concrete implementation
+     * and will queue it inside connection operation queue. When ready to execute, the
+     * {@link RxBleRadioOperationCustom#execute(BluetoothGatt, RxBleGattCallback, Scheduler, Observer)} will be
+     * invoked.
+     * <p>
+     * The operation will be added to the queue using a {@link com.polidea.rxandroidble.internal.RxBleRadioOperation.Priority#NORMAL}
+     * priority.
+     *
+     * @param operation The builder used to create the {@link RxBleRadioOperation} instance
```

Comment:
Sorry, should have warned you that I did not update everything because I didn't want to invest time updating everything if `Observable<T>` was "rejected". I will fix all issues.